src/components/IncrementButton/IncrementButton.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { addUsage } from '../../actions/items';
+
+class IncrementButton extends Component {
```

Comment:
@ludviglundgren, I think we can solve this by binding the addUsage action creator to the Item component instead :smile: